board/recalbox/fsoverlay/recalbox/scripts/recalbox-includes.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+#
+## This is a knid of header file that can be used to declare variables and functions
+## That could turn useful in any recalbox script
+#
+
+#
+## Variables
+#
+
+_RBX=/recalbox
+_SHAREINIT=$_RBX/share_init
+_SHARE=$_RBX/share
+
+#
+## Functions
+#
+
+# Checks if $1 exists in the array passed for $2
+function containsElement {
+  # local e
+  # for e in "${@:2}"; do [[ "$e" == "$1" ]] && return 0; done
+  [[ "${@:2}" =~ "$1" ]] && return 0
+  return 1
+}
+
+# Upgrade the recalbox.conf if necessary
+function doRbxConfUpgrade {
+  # Update recalbox.conf
+  cfgIn=$_SHAREINIT/system/recalbox.conf
+  cfgOut=$_SHARE/system/recalbox.conf
+  rbxVersion=$_RBX/recalbox.version
+  curVersion=$_SHARE/system/logs/lastrecalbox.conf.update
+  updVersion=$_RBX/recalbox.updateversion
```

Comment:
This file does not exist.
We already are on the new version as the update has already be done.
We can remove this file.